packages/cliqz-home-react/src/AddSpeedDial.js

```diff
@@ -0,0 +1,123 @@
+import React, { Component } from 'react';
```

Comment:
would rather a call a component with a noun, not a verb, for example `new-speed-dial-form`